src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Diagnostics/DocumentPullDiagnosticsEndpoint.cs

```diff
@@ -37,13 +48,16 @@ internal class DocumentPullDiagnosticsEndpoint : IRazorRequestHandler<VSInternal
         _telemetryReporter = telemetryReporter;
     }
 
+    /// <inheritdoc/>
```

Comment:
Nit: Same as above, you don't need to specify empty `<inheritdoc />`, the IDE will assume they're there automatically, so these are just unnecessary lines.